src/main/java/uk/gov/register/thymeleaf/ThymeleafView.java

```diff
@@ -94,6 +94,10 @@ public String getScheme() {
         return requestContext.getScheme();
     }
 
+    @SuppressWarnings("unused, used by templates")
+    public Boolean getIsGovukBranded() {
+        return requestContext.getHost().endsWith("register.gov.uk");
```

Comment:
discussed IRL with @samcrang - this is probably fine for now, especially since this is a temporary solution.

the [host header definition](https://tools.ietf.org/html/rfc7230#section-5.4) says that the port should be specified if it's not the default port, but doesn't say if it might be specified if it is the default port.